docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   db:
-    container_name: postgres-db
+    container_name: take-home-db
```

Comment:
This is a reasonable change. 👍  Can you please reset your PR to have this in it but not the column change? Thank you!

(I usually don't bump into this, as I stop all running Docker containers when I move from project to project.)